index.d.ts

```diff
@@ -0,0 +1,27 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+interface MochaGiven {
+    <P1, P2, P3>(...params: { 0: P1, 1: P2, 2: P3 }[]): MochaGivenMethods3<P1, P2, P3>;
+    <P1, P2>(...params: { 0: P1, 1: P2 }[]): MochaGivenMethods2<P1, P2>;
+    <P1>(...params: P1[]): MochaGivenMethods1<P1>;
```

Comment:
I'm fine if we exclude it. The only reason I brought it up was because I had a situation where I defined the test cases as an array and stored it in a variable and then tried to pass that array as a parameter to `given`. It wouldn't compile unless I used the `spread` operator:

```ts
const data = [ { a: 1, b: 2 }, { a: 3, b: 4 } ];

given(data).it("works", (a, b) => { ... }); //=> compile error, types of 'a' and 'b' were incorrectly inferred
given(...data).it("works", (a, b) => { ... }); //=> no error
```